### ISSUE TITLE
occ previews:cleanup babbles while working

### ISSUE THREAD

**Issue by jnweiger** [owncloud/core#41026]
Seen with 10.13.2-rc.1

* create a few hundered images, delete them, delete them from trashbin
* run `occ previews:cleanup` (without any verbose options):
```
352955 - admin: deleted
350618 - admin: deleted
355174 - admin: deleted
349664 - admin: deleted
347205 - admin: deleted
354083 - admin: deleted
351928 - admin: deleted
348419 - admin: deleted
345797 - admin: deleted
352913 - admin: deleted
350596 - admin: deleted
355172 - admin: deleted
349656 - admin: deleted
347179 - admin: deleted
354063 - admin: deleted
351905 - admin: deleted
348406 - admin: deleted
345743 - admin: deleted
352815 - admin: deleted
...
350589 - admin: deleted
355165 - admin: deleted
349537 - admin: deleted
347175 - admin: deleted
353942 - admin: deleted
351904 - admin: deleted
289 orphaned previews deleted
```
* the command produces one line of output per deleted thumbnail. BAD

Expected behaviour: 
* the detailed (mostly useless?)  listing of deleted file-ids is only shown with -vvv

**Comment by pako81** [comment]
Also seen with 10.13.1 - not a regression.

**Comment by pako81** [comment]
Fix proposed in https://github.com/owncloud/core/pull/41030

**Comment by jnweiger** [comment]
Confirmed fixed:
With this one-liner change, 
`occ previews:cleanup -vv` and `occ previews:cleanup -vvv`
report every entry, 
`occ previews:cleanup -v` or `occ previews:cleanup` only report one summary line:

```

9 orphaned previews deleted

```

**Comment by phil-davis** [comment]
fixed in 10.13.2
